problems.h

```diff
@@ -5,9 +5,55 @@ using namespace std;
 // reverseInt(6883) should return 3886
 // reverseInt(0) should return 0
 // reverseInt(-2938947) should return -7498392
+
+vector<Stack<int>> stacks;
+
 int reverseInt(int num)
 {
     // Your code here
+    bool isNegative = 0;
+    int newNum = 0;
+    if (num < 0){
+        isNegative = 1;
+        num = abs(num);
+    }
+    
+    //flipping num first time- num is reversed
+    int placeholder = num;
+    int flipNum = 0;
+    int rem = 0;
+    while (placeholder != 0){
+        flipNum *= 10;
+        rem = placeholder % 10;
+        flipNum += rem;
+        placeholder /= 10;
+    }
+    flipNum *= 10;
+    rem = placeholder % 10;
+    flipNum += rem;
+    num = flipNum;
+    
+    if (num > 0){
+        //flips it again while putting it in stack- num is in orginal order 
+        int remainder = num % 10;
+        Stack<int> s;
+        num /= 10;
+        while (num != 0){
+            remainder = num % 10;
+            s.push(remainder);
+            num /= 10;
+        }
+
+        //flips it again while taking nums out- num is reversed again
+        while (!s.isEmpty()){
```

Comment:
why didn't you just multiply the popped number by 10 and then add? Would have avoided flipping twice!